test/data/countries.js

```diff
@@ -0,0 +1,14 @@
+import { expect } from 'chai';
```

Comment:
I could definitely add more tests for this module since it also includes a function to return translations, but it depends on app.polyglot and I'm not quite sure how to stub that in. Maybe something like Sinon can help with that. I suppose I may just be able to add Polyglot to the app module inside of the test... but I'm not sure what the best approach is right now, so I'll have to circle back around to this.